cli/create/templates/cartridge/systemd-unit-params.yml

```diff
@@ -0,0 +1,5 @@
+---
+# Here you can specify default parameters for unit files, such as:
+
+# fd-limit: maximum-number-of-fd
```

Comment:
# fd-limit: LimitNOFILE for application instance
# stateboard-fd-limit: LimitNOFILE for stateboard instance